demo/src/main/AndroidManifest.xml

```diff
@@ -8,8 +8,17 @@
         android:label="@string/app_name"
         android:theme="@style/AppTheme" >
         <activity
-            android:name=".PinDemoActivity"
-            android:label="@string/app_name" >
+                android:name=".PinDemoActivity"
+                android:label="@string/app_name" >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
```

Comment:
The problem is that the Activity is either a **Activity** or a **FragmentActivity** in order to get the right Fragment manager.